The Linux failure:

```
FAIL: Swift(linux-x86_64) :: Python/build_swift.swift (8145 of 11792)
******************** TEST 'Swift(linux-x86_64) :: Python/build_swift.swift' FAILED ********************
Script:
--
: 'RUN: at line 10';   /usr/bin/python -m unittest discover -s /home/buildnode/jenkins/workspace/swift-PR-Linux-smoke-test/branch-master/swift/utils/build_swift/ -t /home/buildnode/jenkins/workspace/swift-PR-Linux-smoke-test/branch-master/swift/utils/
--
Exit Code: 1

Command Output (stderr):
--
....................................................................................................................................................................................................................................................................................................................................................F..........................................................................................................................................................................FF........................................................................................................................................................................................
======================================================================
FAIL: test_default_value_lldb_build_with_xcode (build_swift.tests.test_driver_arguments.buildbot_incremental,tools=RA,stdlib=RA)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/buildnode/jenkins/workspace/swift-PR-Linux-smoke-test/branch-master/swift/utils/build_swift/tests/test_driver_arguments.py", line 126, in test
    .format(dest, default_value, parsed_value))
AssertionError: Invalid default value for "lldb_build_with_xcode": 1 != 0

======================================================================
FAIL: test_option_lldb_build_with_cmake (build_swift.tests.test_driver_arguments.buildbot_incremental,tools=RA,stdlib=RA)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/buildnode/jenkins/workspace/swift-PR-Linux-smoke-test/branch-master/swift/utils/build_swift/tests/test_driver_arguments.py", line 144, in test
    self.assertEqual(getattr(namespace, option.dest), option.value)
AssertionError: '1' != '0'

======================================================================
FAIL: test_option_lldb_build_with_xcode (build_swift.tests.test_driver_arguments.buildbot_incremental,tools=RA,stdlib=RA)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/buildnode/jenkins/workspace/swift-PR-Linux-smoke-test/branch-master/swift/utils/build_swift/tests/test_driver_arguments.py", line 144, in test
    self.assertEqual(getattr(namespace, option.dest), option.value)
AssertionError: '0' != '1'

----------------------------------------------------------------------
Ran 697 tests in 16.106s

FAILED (failures=3)

--

********************
```

--- a/utils/build-presets.ini
+++ b/utils/build-presets.ini
@@ -528,7 +528,7 @@ assertions
 test
 validation-test
 
-llvm-targets-to-build=X86;AArch64;PowerPC
+llvm-targets-to-build=X86;ARM;AArch64;PowerPC
 
 # Set the vendor to apple
 compiler-vendor=apple

--- a/utils/build-script-impl
+++ b/utils/build-script-impl
@@ -74,7 +74,7 @@ KNOWN_SETTINGS=(
     swift-stdlib-enable-assertions "1"           "enable assertions in Swift"
     swift-stdlib-use-nonatomic-rc "0"            "build the Swift stdlib and overlays with nonatomic reference count operations enabled"
     lldb-build-type             "Debug"          "the CMake build variant for LLDB"
-    lldb-build-with-xcode       "1"              "Use xcodebuild to build LLDB, instead of CMake"
+    lldb-build-with-xcode       "0"              "Use xcodebuild to build LLDB, instead of CMake"
     llbuild-build-type          "Debug"          "the CMake build variant for llbuild"
     foundation-build-type       "Debug"          "the build variant for Foundation"
     libdispatch-build-type      "Debug"          "the build variant for libdispatch"

--- a/utils/build_swift/driver_arguments.py
+++ b/utils/build_swift/driver_arguments.py
@@ -77,7 +77,7 @@ def _apply_default_arguments(args):
         args.lldb_build_variant = args.build_variant
 
     if args.lldb_build_with_xcode is None:
-        args.lldb_build_with_xcode = '1'
+        args.lldb_build_with_xcode = '0'
 
     if args.foundation_build_variant is None:
         args.foundation_build_variant = args.build_variant
@@ -629,11 +629,11 @@ def create_argument_parser():
            help='build the Debug variant of LLDB')
 
     option('--lldb-build-with-xcode', store('lldb_build_with_xcode'),
-           const='1',
+           const='0',
            help='build LLDB using xcodebuild, if possible')
 
     option('--lldb-build-with-cmake', store('lldb_build_with_xcode'),
-           const='0',
+           const='1',
            help='build LLDB using CMake')
 
     option('--debug-cmark', store('cmark_build_variant'),

--- a/utils/build_swift/driver_arguments.py
+++ b/utils/build_swift/driver_arguments.py
@@ -629,11 +629,11 @@ def create_argument_parser():
            help='build the Debug variant of LLDB')
 
     option('--lldb-build-with-xcode', store('lldb_build_with_xcode'),
-           const='0',
+           const='1',
            help='build LLDB using xcodebuild, if possible')
 
     option('--lldb-build-with-cmake', store('lldb_build_with_xcode'),
-           const='1',
+           const='0',
            help='build LLDB using CMake')
 
     option('--debug-cmark', store('cmark_build_variant'),

--- a/utils/build_swift/tests/expected_options.py
+++ b/utils/build_swift/tests/expected_options.py
@@ -152,7 +152,7 @@ EXPECTED_DEFAULTS = {
     'llbuild_assertions': True,
     'lldb_assertions': True,
     'lldb_build_variant': 'Debug',
-    'lldb_build_with_xcode': '1',
+    'lldb_build_with_xcode': '0',
     'llvm_assertions': True,
     'llvm_build_variant': 'Debug',
     'llvm_max_parallel_lto_link_jobs':

